src/main/java/org/spongepowered/common/data/util/DataConstants.java

```diff
@@ -105,6 +106,13 @@ private DataConstants() {}
     // Original (0) / Copy of original (1) / Copy of a copy (2) / Tattered (3)
     public static final int MAXIMUM_GENERATION = 3;
 
+    public static final Vector3d DEFAULT_HEAD_ROTATION = new Vector3d(0, 0, 0);
```

Comment:
would be a good idea to link these up to the `EntityArmorStand.DEFAULT_HEAD_ROTATION` field so it can remain inline with vanilla